Translations-iOS/cs.xcloc/Localized Contents/cs.xliff

```diff
@@ -335,15 +335,15 @@
         <note>No comment provided by engineer.</note>
       </trans-unit>
       <trans-unit id="%lld seconds" xml:space="preserve">
-        <source>%lld seconds</source>
+        <source>%lld sekund</source>
```

Comment:
Thanks a lot again @ferben! For future updates the proper way to update this file (manually) is by adding the `<target>` key with the translation, like the following:

```xlm
        <source>%lld seconds</source>
        <target>%lld sekund</target>
```

Anyway you only need to update the `xliff` or the strings files, not both. Here's the final `xliff` with your updated translations: https://github.com/aonez/Keka/commit/5af4b31b815e564714de9d2f70cb64f64b977c45